glauth/docs/getting-started.md

```diff
@@ -0,0 +1,272 @@
+
+### Docker
+
+TBD
+
+### Binaries
+
+TBD
+
+## Configuration
+
+We provide overall three different variants of configuration. The variant based on environment variables and commandline flags are split up into global values and command-specific values.
+
+### Envrionment variables
+
+If you prefer to configure the service with environment variables you can see the available variables below.
+
+#### Global
+
+GLAUTH_CONFIG_FILE
+: Path to config file, empty default value
+
+GLAUTH_LOG_LEVEL
+: Set logging level, defaults to `info`
+
+GLAUTH_LOG_COLOR
+: Enable colored logging, defaults to `true`
+
+GLAUTH_LOG_PRETTY
+: Enable pretty logging, defaults to `true`
+
+#### Server
+
+GLAUTH_TRACING_ENABLED
+: Enable sending traces, defaults to `false`
+
+GLAUTH_TRACING_TYPE
+: Tracing backend type, defaults to `jaeger`
+
+GLAUTH_TRACING_ENDPOINT
+: Endpoint for the agent, empty default value
+
+GLAUTH_TRACING_COLLECTOR
+: Endpoint for the collector, empty default value
+
+GLAUTH_TRACING_SERVICE
+: Service name for tracing, defaults to `glauth`
+
+GLAUTH_DEBUG_ADDR
+: Address to bind debug server, defaults to `0.0.0.0:9124`
+
+GLAUTH_DEBUG_TOKEN
+: Token to grant metrics access, empty default value
+
+GLAUTH_DEBUG_PPROF
+: Enable pprof debugging, defaults to `false`
+
+GLAUTH_DEBUG_ZPAGES
+: Enable zpages debugging, defaults to `false`
+
+GLAUTH_HTTP_ADDR
+: Address to bind http server, defaults to `0.0.0.0:9120`
+
+GLAUTH_HTTP_NAMESPACE
+: The http namespace
+
+GLAUTH_HTTP_ROOT
+: Root path of http server, defaults to `/`
+
+#### Health
+
+GLAUTH_DEBUG_ADDR
+: Address to debug endpoint, defaults to `0.0.0.0:9124`
+
+### Commandline flags
+
+If you prefer to configure the service with commandline flags you can see the available variables below.
+
+#### Global
+
+--config-file
+: Path to config file, empty default value
+
+--log-level
+: Set logging level, defaults to `info`
+
+--log-color
+: Enable colored logging, defaults to `true`
+
+--log-pretty
+: Enable pretty logging, defaults to `true`
+
+#### Server
+
+--tracing-enabled
+: Enable sending traces, defaults to `false`
+
+--tracing-type
+: Tracing backend type, defaults to `jaeger`
+
+--tracing-endpoint
+: Endpoint for the agent, empty default value
+
+--tracing-collector
+: Endpoint for the collector, empty default value
+
+--tracing-service
+: Service name for tracing, defaults to `glauth`
+
+--debug-addr
+: Address to bind debug server, defaults to `0.0.0.0:9124`
+
+--debug-token
+: Token to grant metrics access, empty default value
+
+--debug-pprof
+: Enable pprof debugging, defaults to `false`
+
+--debug-zpages
+: Enable zpages debugging, defaults to `false`
+
+--http-addr
+: Address to bind http server, defaults to `0.0.0.0:9120`
+
+--http-namespace
+: Namespace for internal services communication, defaults to `com.owncloud.web`
+
+--http-root
+: Root path of http server, defaults to `/`
+
+#### Health
+
+--debug-addr
+: Address to debug endpoint, defaults to `0.0.0.0:9124`
+
```

Comment:
Codacy found an issue: [[no-duplicate-headings] Do not use headings with similar content (46:1)](https://app.codacy.com/gh/owncloud/ocis/pullRequest?prid=6202733)